test/Microsoft.Azure.SignalR.Management.Tests/UnitTests/ServiceManagerFacts.cs

```diff
@@ -67,5 +70,14 @@ internal void GenerateClientEndpointTest()
 
             Assert.Equal(_clientEndpoint, clientEndpoint);
         }
+
+        [Fact]
+        internal void GenerateClientEndpointWithAppNameTest()
+        {
+            var manager = new ServiceManager(new ServiceManagerOptions() { ConnectionString = _testConnectionString, ApplicationName = _appName });
```

Comment:
Could you apply the `ApplicationName` to all other tests in this file?
Simply add `ApplicationName` in _serviceManagerOptions  (line 23)?